docs/getting-started/release_package.md

```diff
@@ -0,0 +1,142 @@
+      https://github.com/kubeedge/kubeedge/blob/master/build/crds/devices/devices_v1alpha1_device.yaml
+      kubectl create -f devices_v1alpha1_device.yaml
+     ```    
+  + Run cloud
+  
+      ```shell
+      cd /etc/kubeedge/cloud
+      # run edge controller
+      # `conf/` should be in the same directory where edgecontroller resides
+      # verify the configurations before running cloud(edgecontroller)
+      ./edgecontroller
+      ```
+  ## Edge Vm   
+   **NOTE:** scp kubeedge folder from cloud vm to edge vm
+   ```
+   In cloud
+   scp -r /etc/kubeedge root@edgeip:/etc
+   ```
+   ### Configuring MQTT mode
+   
+   The Edge part of KubeEdge uses MQTT for communication between deviceTwin and devices. KubeEdge supports 3 MQTT modes:
+   1) internalMqttMode: internal mqtt broker is enabled.
+   2) bothMqttMode: internal as well as external broker are enabled.
+   3) externalMqttMode: only external broker is enabled.
+   
+   Use mode field in [edge.yaml](etc/kubeedge/edge/conf/edge.yaml#L4) to select the desired mode.
+   
+   To use KubeEdge in double mqtt or external mode, you need to make sure that [mosquitto](https://mosquitto.org/) or [emqx edge](https://www.emqx.io/downloads/edge) is installed on the edge node as an MQTT Broker.
+   
+   + We have provided a sample node.json to add a node in kubernetes. Please make sure edge-node is added in kubernetes. Run below steps to add edge-node.
+   
+   + Modify the `https://github.com/kubeedge/kubeedge/blob/master/build/node.json` file and change `metadata.name` to the name of the edge node
+   
+   **Note**:node.json must be copied to any specific folder, then run kubectl command as shown below
+   
+   + Deploy node
+       ```shell
+       kubectl apply -f node.json
+       ```
+   + Modify the `/etc/kubeedge/edge/conf/edge.yaml` configuration file
+       + Replace `edgehub.websocket.certfile` and `edgehub.websocket.keyfile` with your own certificate path
+       + Update the IP address of the master in the `websocket.url` field. 
+       + replace `fb4ebb70-2783-42b8-b3ef-63e2fd6d242e` with edge node name in edge.yaml for the below fields :
+           + `websocket:URL`
+           + `controller:node-id`
+           + `edged:hostname-override`
+    
+   + Run edge   
+       ```
+       # run edge_core
+           # `conf/` should be in the same directory as the cloned KubeEdge repository
+           cd /etc/kubeedge/edge
+           # verify the configurations before running edge(edge_core)
+           ./edge_core
+           # or
+           nohup ./edge_core > edge_core.log 2>&1 &
+          
+      ```
+    **Note**: Running edge_core on ARM based processors,follow the above steps as mentioned for Edge Vm
+    ```
+     wget https://github.com/kubeedge/kubeedge/releases/download/-$VERSION/kubeedge-$VERSION-$OS-$ARCH.tar.gz
+     ex:https://github.com/kubeedge/kubeedge/releases/download/v0.3.0/kubeedge-v0.3.0-linux-arm.tar.gz    
+              
+    ```
+    
```

Comment:
Please remove the unnecessary empty lines.

Reply:
ok